posthog/temporal/data_imports/pipelines/stripe/__init__.py

```diff
@@ -266,7 +267,9 @@ def stripe_source(
             },
             "headers": {
                 "Stripe-Account": account_id,
-            },
+            }
+            if account_id
+            else None,
```

Comment:
If `account_id`s can be empty strings, this would be invalid as `None` is falsy. I don't think Stripe allows empty string account ids, though, right?